src/main/java/net/dv8tion/jda/api/entities/Activity.java

```diff
@@ -308,7 +334,13 @@ static boolean isValidStreamingUrl(@Nullable String url)
          * @incubating This feature is currently not officially documented and might change
          */
         @Incubating
-        CUSTOM_STATUS(4);
+        CUSTOM_STATUS(4),
+
+        /**
+         * Used to indicate that the {@link Activity Activity} should display
+         * as {@code Competing in...} in the official client.
+         */
+        COMPETING(5);
```

Comment:
Why is this not marked with @Incubating as well, if the builder method is? Should be added to both for consistency.

Reply:
It is generally confusing as the other ones that have this annotation have the reason, that it isn't documented, despite it being documented... Just not fully.
I explained this in my initial PR comment.